.travis.yml

```
@@ -2,10 +2,9 @@ language: bash
 services: docker
 
 env:
-  - POSTGIS_MAJOR=2.2 POSTGIS_VERSION=2.2.2+dfsg-7~137.gite11228b.pgdg80+1 PG_MAJOR=9.5 VARIANT=slim
-  - POSTGIS_MAJOR=2.3 POSTGIS_VERSION=2.3.3+dfsg-1.pgdg80+1 PG_MAJOR=9.5 VARIANT=slim
-  - POSTGIS_MAJOR=2.3 POSTGIS_VERSION=2.3.3+dfsg-1.pgdg80+1 PG_MAJOR=9.6 VARIANT=slim
-  - POSTGIS_MAJOR=2.4 POSTGIS_VERSION=2.4.3+dfsg-2.pgdg90+1 PG_MAJOR=10.3 VARIANT=slim
+  - POSTGIS_MAJOR=2.3 POSTGIS_VERSION=2.3.3+dfsg-1.pgdg90+1 PG_MAJOR=9.5 VARIANT=slim
+  - POSTGIS_MAJOR=2.3 POSTGIS_VERSION=2.3.3+dfsg-1.pgdg90+1 PG_MAJOR=9.6 VARIANT=slim
+  - POSTGIS_MAJOR=2.4 POSTGIS_VERSION=2.4.4+dfsg-4.pgdg90+1 PG_MAJOR=10.3 VARIANT=slim
 
```

I forget how, but I recall that we were able to prove that Amazon RDS allows you to create PostgreSQL 10.6 instances. We should add a `PG_MAJOR=10.6` variant to the build matrix too.

```suggestion
  - POSTGIS_MAJOR=2.4 POSTGIS_VERSION=2.4.4+dfsg-4.pgdg90+1 PG_MAJOR=10.6 VARIANT=slim
  
```